lib/point_quest_web/live/quest.ex

```diff
@@ -163,8 +183,8 @@ defmodule PointQuestWeb.QuestLive do
   defp is_party_leader?(%PartyLeader{} = _actor), do: true
   defp is_party_leader?(_actor), do: false
 
-  defp show_attack_panel?(%PartyLeader{adventurer: nil}), do: false
-  defp show_attack_panel?(_actor), do: true
+  defp show_attack_panel?(%PartyLeader{adventurer: nil}, _round_active?), do: false
+  defp show_attack_panel?(_actor, round_active?), do: round_active?
```

Comment:
Just commenting here for posterity: we discussed this during implementation and we'll eventually refactor this to disable the attack buttons instead of hiding the controls.

Reply:
true